retriever/lib/store.js

```diff
@@ -92,33 +92,43 @@ export async function getOwnerByRootCid(env, rootCid) {
    LEFT OUTER JOIN indexer_proof_sets ips
      ON ir.set_id = ips.set_id
    WHERE ir.root_cid = ?
-   LIMIT 1;
  `
 
-  /** @type {{ set_id: string; owner: string | null } | null} */
-  const result = await env.DB.prepare(query).bind(rootCid).first()
+  const result = await (await env.DB.prepare(query).bind(rootCid).all()).results
```

Comment:
```suggestion
  const results = await (await env.DB.prepare(query).bind(rootCid).all()).results
```

Should be called `results`, as it is an array now. Btw, there are too many `awaits` here, it should be like this then:

```suggestion
  const results = (await env.DB.prepare(query).bind(rootCid).all()).results
```

And simplifier with more modern syntax:

```suggestion
  const { results } = await env.DB.prepare(query).bind(rootCid).all()
```

Reply:
Addressed in 567a17c